README.md

```diff
@@ -209,6 +209,15 @@ jupyter notebook
 - Get the IP address of your computer by running `ip address show | grep "inet " | grep -v 127.0.0.1` if using Linux/Mac. For windows there are different steps. Alternatively, if you want to run the demo app in the emulator, use `10.0.2.2` as the IP address.
 - Use this IP address and the port (default:5000) in your login screen to supply the PyGrid server url, e.g., 10.0.2.2:5000
 
+### Try the wearable app
+
```

Comment:
The title should be something generic for example,

`Built on KotlinSyft`

Reply:
cool thanks